web/components/input/buttons/CheckboxGroup/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const InputContext = React.createContext();
+
+const Checkbox = ({ name, value, label, checked, disabled, onChange }) => (
+  <InputContext.Consumer>
+    {context => (
+      <label className="checkbox">
+        <input
+          type="checkbox"
+          name={name || context.name}
+          value={value}
```

Comment:
Why do we need `value`? Couldn't `onChange` just supply the value in the callback?

Reply:
Not sure if I follow. Normally in regular forms you can have an input without an `onChange` function (the value is passed on form submit) and so I wanted to support that use case. I haven't played around with forms in React before, is there a reason to have the `onChange` without the `value`?

Reply:
I think this goes for any checkbox form - checkbox is just a toggle, `onChange` just needs to notify if value is `true` or `false`

Reply:
OH SHOOT, I've mixed up my logic, the shared `name` prop is only useful to radio buttons (where there is only a single answer), hence why `value` is there so you can distinguish between radio button options. 

But this is a checkbox... :skull: , I'll fix all of this tomorrow.

Reply:
That's fair actually, I think you're right